### ISSUE TITLE
The windows setups don't work

### ISSUE THREAD

**Issue by LexXi910** [DrewThomasson/VoxNovel#25]
Constantly running into errors and going in circles. Had to ask 2 other friends for help with this and we didn't come to a conclusion. Needs an updated and more specific guide.

**Comment by EdouardVuillard** [comment]
I agree with LexXi910.  Here are some observations in case they help someone to get further.
(1) I ran the commands in the "In Depth" section in addition to running the "Easy Windows 11 install". This seemed to get me further through the install process but not all the way.  Note that you need to take care to separate the blancmange of multiple commands into separate commands, which is not entirely clear from the way they are formatted.
(2) When called upon to close and re-open shell by a message in the terminal, you need to close the powershell window, restart powershell, then type **wsl** to get back to the Linux shell
(3) Paragraph 3 of the commands listed in "In Depth" refer to a folder **miniconda3**.  Despite following all other instructions to the letter, that folder did not exist but **miniconda** did, at **/home/(username)/miniconda**.  That is where I found the executable file **_conda**.
(4) When called upon to type any command where **conda** is the executable, I navigated to the folder that contains **_conda** and substituted **_conda** for **conda**.  Also, I added **./** before **_conda**.
(5) I was able to get the commands up to and including the first command (the command that starts with **conda create**) in paragraph 3 of "In Depth" to work.  However, **conda activate VoxNovel** produced an error to the effect that I needed to run **conda init** first.  Running **conda init** produced the warning _Cannot install xonsh wrapper without a python interpreter in prefix_ followed by a series of errors that were beyond my knowledge to address.

**Comment by DrewThomasson** [comment]
Hey, sorry about the late response,

@EdouardVuillard, you are absolutely right about the instructions being poorly made.

I've been thinking about removing the "In Depth" Windows install part because it seems to be causing more confusion than solving problems.

All you honestly need is the single command install in the easy commands section. It should do everything for you as long as you're in a new, blank WSL environment. If you ever have any issues, you should just unregister your Ubuntu WSL environment, reinstall Ubuntu in WSL, and try again with the single command install. That seems to fix any issues.

What do you guys think? Should I remove the "In Depth" install part or label it as deprecated?

## Current Testing in progress...

Also, I am currently creating a new blank WSL environment to reinstall everything and check for any issues on my end. 

Thank you SO MUCH for the feedback and I would love any suggestions on the readme or anything.

**Comment by DrewThomasson** [comment]
## Update 1

Just made the "InDepth install" Deprecated 

So it is now labeled as

"InDepth Windows 11 install (DEPRECATED-DO NOT USE, for development purposes only)"

## In progress Currently updating the windows single command install script and readme guide

**Comment by DrewThomasson** [comment]
# I'll be releasing a Windows install video soon enough

**Comment by DrewThomasson** [comment]
## Windows 11 install video

https://youtu.be/OmJub3uvfz4

**Comment by EdouardVuillard** [comment]
Thanks Drew.  I really appreciate your work.  As suggested, I started afresh.  I unregistered the Ubuntu installation using **wsl --unregister Ubuntu** and then started again using your "easy" instructions.  There were a couple or so warnings in red that zipped past but in the end the install worked.  The **cd VoxNovel** instruction in the "easy" instructions assumes that the previous commands left the user in their home directory which was not the case for me.  I worked out where it is but I note that for many non-Linux users it might not be obvious that the VoxNovel directory is in the home directory so perhaps the final **cd** instruction could include the explicit fully qualified path.  (Bear in mind that to a Windows user, even the commonly used **~** could be a mystery).  I have now successfully launched the GUI and started converting my first book, which has led to another issue but I will start a separate thread for that.
